### ISSUE TITLE
DBeaver fails to start when command line arguments are supplied

### ISSUE THREAD

**Issue by MorpheusXAUT** [dbeaver/dbeaver#14370]
#### System information: 
- Windows 10 Pro 64-bit (10.0, Build 19042)
- DBeaver version: 21.2.3.202110151816

#### Describe the problem you're observing:
Supplying command line arguments (in my specific case `-closeTabs` and `-disconnectAll`) causes DBeaver to show its splash screen briefly, then exits without actually starting up. This seems to be the case with all command line arguments I tried (including `-help`) - removing all arguments leads to DBeaver launching normally.
This issue seems to exist since the last update as previous versions worked just fine with the mentioned command line arguments.
A colleague of mine seemed to be able to replicate the issue on Mac OS 11.6 as well.

#### Steps to reproduce, if exist:
- Update to latest version of DBeaver currently available
- Start DBeaver via shortcut/command line and supply at least one argument

#### Include any warning/errors/backtraces from the logs
No relevant logs exist in the workspace metadata folder.
`<install-path>/configuration` contains the following log file:
```
!SESSION 2021-10-25 10:53:41.743 -----------------------------------------------
eclipse.buildId=unknown
java.version=11.0.12
java.vendor=Eclipse Foundation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en
Framework arguments:  -closeTabs -disconnectAll
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -closeTabs -disconnectAll

!ENTRY org.jkiss.dbeaver.model 2 0 2021-10-25 10:53:42.929
!MESSAGE Can't connect to running instance, exiting
```

**Comment by uslss** [comment]
thanks for the report.
will be fixed in 21.2.4 (#14284)

**Comment by MorpheusXAUT** [comment]
Sorry for the duplicate report, unfortunately I didn't find the linked issue after a brief search. Glad to hear it's being fixed! 👍
